src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -277,17 +248,19 @@ bool InstanceKlass::has_as_permitted_subclass(const InstanceKlass* k) const {
 // value of it and, if applicable, the nest host resolution/validation error,
 // are idempotent.
 InstanceKlass* InstanceKlass::nest_host(TRAPS) {
+  JavaThread* current = THREAD->as_Java_thread();
```

Comment:
I don't see why this is necessary to do this conversion.

Reply:
So that we don't use THREAD in the call to has_nest_member as it is not exception related. But if the Thread arg goes first on that call I can switch this back.